ipn/ipnlocal/c2n.go

```diff
@@ -59,6 +60,14 @@ func (b *LocalBackend) handleC2N(w http.ResponseWriter, r *http.Request) {
 		} else {
 			http.Error(w, "no log flusher wired up", http.StatusInternalServerError)
 		}
+	case "/posture/identity":
+		switch r.Method {
+		case http.MethodGet:
```

Comment:
httpm.GET or just "GET"

See https://pkg.go.dev/tailscale.com/util/httpm for the backstory.

Or, uh, I see they've crept in otherwise on line 45 etc.

I can go fix them all later & add a test to stop the bleeding since that's my pet peeve.

Reply:
I can fix this and the other ones in a follow up if you dont beat me too it.